src/azure-cli/azure/cli/command_modules/appconfig/_kv_helpers.py

```diff
@@ -283,7 +283,8 @@ def __write_kv_and_features_to_file(file_path, key_values=None, features=None, f
             elif format_ == 'yaml':
                 yaml.safe_dump(exported_keyvalues, fp, sort_keys=False, width=float('inf'))
             elif format_ == 'properties':
-                javaproperties.dump(exported_keyvalues, fp)
+                for key, value in exported_keyvalues.items():
+                    fp.write('{}={}\n'.format(key, value))
```

Comment:
```suggestion
                    fp.write('{}={}\n'.format(javaproperties.escape(key), value))
```

Reply:
@SaurabhSharma-MSFT Thank you for the response and my apologies for the late reply. The response is well understood, however, since there may be some values in the key that may be considered as a separator, we should still escape the key so it is not parsed differently in the properties file. We can keep the value property as-is. 

@mrm9084 @avanigupta Could we please get your thoughts on this, and whether the proposed change is sufficient?

Reply:
I agree. We should allow unescaped special characters in the value (which is what the customer is interested in) and escape special characters in the key. 

@albertofori can you help to verify if importing unescaped values might cause problems after this change?
Also, is the behavior same in portal?

Reply:
@avanigupta The only difference that will be observed is that leading white spaces will be trimmed off values. e.g., For the following key-values in a properties file format:
![image](https://github.com/Azure/azure-cli/assets/102393878/2537beb6-c2e2-46fd-9761-691408bedf98)

The imported key-values will look as shown below:
![image](https://github.com/Azure/azure-cli/assets/102393878/315afd8a-aaaf-4418-9daa-325ec2ab5bd6)

The key `my:url` has the 2 leading spaces of the value trimmed. Trailing spaces are maintained though, as can be seen with the value of key `my`.
In order to maintain leading spaces, we can escape the first leading space.

Reply:
Does portal have the same behavior? 
I think this is the right behavior because leading whitespaces in the value will be ignored by other parsing libraries too. This will break our round-trip capability, but leading whitespaces in properties file values are not a valid scenario anyway.

Reply:
Sounds good, we can update the portal to escape keys. Thanks, Albert!

Reply:
@SaurabhSharma-MSFT Could you please, take a look at and accept the suggested changes to this PR?

Reply:
Done